### PR TITLE
Examples fix / improvement

### PR DESCRIPTION
**Pull request by MoonE**
- Use the same jquery version (3.5.1) in all examples.
- Don't add bootstrap js twice
- Remove unused extraResources in build process
- fix tooltip in kml-earthquake example

### CODE REVIEW

**Comment by MoonE** [comment]
bootstrap js requires jquery and it is used to display the list of other examples for a tag.

File: examples/webpack/example-builder.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ const readFile = promisify(fs.readFile);
 const isCssRegEx = /\.css$/;
 const isJsRegEx = /\.js(\?.*)?$/;
 const importRegEx = /^import .* from '(.*)';$/;
+const isTemplateJs = /\/(jquery|(bootstrap(\.bundle)?))(\.min)?\.js$/;
```
Comment (ahocevar) [inline examples/webpack/example-builder.js:15]: To find the jquery scripts as we include them, the regex should look like this:
```suggestion
const isTemplateJs = /\/(jquery(-[0-9.]+)?|(bootstrap(\.bundle)?))(\.min)?\.js$/;
```

**Comment by MoonE** [comment]
Please merge, thanks for asking.

**Comment by jipexu** [comment]
thanks !

**Pull request merged**